### PR TITLE
Adding README to Josephus Problem

### PR DESCRIPTION
**Pull request by rimshaakhan**
**Fixes issue:**
N/A

**Changes:**
Created README file for Josephus Problem.

### CODE REVIEW

File: code/unclassified/src/josephus_problem/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Josephus Problem  
+
+Josephus Problem is theoretical Counting Game Problem.
```
Comment (abdouskamel) [inline code/unclassified/src/josephus_problem/README.md]: Josephus problem is **a**...

File: code/unclassified/src/josephus_problem/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Josephus Problem  
+
+Josephus Problem is theoretical Counting Game Problem.  
+n number of person stand in a circle and are numbered from 1 to n in clockwise direction.  
+Starting from 1 in clockwise direction each person kills k<sup>th</sup> person next to him and this continues till
```
Comment (abdouskamel) [inline code/unclassified/src/josephus_problem/README.md]: kills **the** k<sup>th</sup>...

File: code/unclassified/src/josephus_problem/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+a single person is left. The problem is to choose a suitable postion in initial circle to avoid execution.
+
+## Solution
+
+Let J(n,k) represent the position of survivor when there are n person and k denotes count of each step i.e. k-1 person 
+are skipped and k<sup>th</sup> person is executed.
+
+### Recurrence Relation  
+`J(1,k) = 1`
```
Comment (abdouskamel) [inline code/unclassified/src/josephus_problem/README.md]: The newline isn't appearing here.

File: code/unclassified/src/josephus_problem/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Josephus Problem  
+
+Josephus Problem is theoretical Counting Game Problem.  
+n number of person stand in a circle and are numbered from 1 to n in clockwise direction.  
+Starting from 1 in clockwise direction each person kills k<sup>th</sup> person next to him and this continues till
+a single person is left. The problem is to choose a suitable postion in initial circle to avoid execution.
+
+## Solution
+
+Let J(n,k) represent the position of survivor when there are n person and k denotes count of each step i.e. k-1 person
```
Comment (abdouskamel) [inline code/unclassified/src/josephus_problem/README.md]: Please rewrite this part to make it more clear.

**Comment by rimshaakhan** [comment]
@abdouskamel Can you please review the changes?

File: code/unclassified/src/josephus_problem/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Josephus Problem is a theoretical Counting Game Problem.  
+n people stand in a circle and are numbered from 1 to n in clockwise direction.  
+Starting from 1 in clockwise direction each person kills the k<sup>th</sup> person next to him and this continues till
+a single person is left. The task is to choose a suitable position in initial circle to avoid execution.
+
+## Explaination
+
+For example, if n = 5 and k = 2, then the safe position is 3. Firstly, the person at position 2 is executed, then person at position 4 is executed, then person at position 1 is killed. Finally, the person at position 5 is executed. So the person at position 3 survives.  
+
+If n = 7 and k = 3, then the safe position is 4. The persons at positions 3, 6, 2, 7, 5, 1 are executed in order, and person at position 4 survives.
```
Comment (abdouskamel) [inline code/unclassified/src/josephus_problem/README.md]: ... and **the** person at ...

File: code/unclassified/src/josephus_problem/README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+Let J(n,k) represent the position of survivor when there are n person and k denotes count of each step i.e. k-1 person are skipped and k<sup>th</sup> person is executed.
```
Comment (abdouskamel) [inline code/unclassified/src/josephus_problem/README.md]: of **the** survivor

Comment (abdouskamel) [inline code/unclassified/src/josephus_problem/README.md]: and **the** k<sup>th</sup

File: code/unclassified/src/josephus_problem/README.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+We observe that after the first kill i.e. the killing of second person we have n-1 people so problem can be reduced to find safest position with n-1 people when person on position k is executed.
+
+### Recurrence Relation 
+
+`J(1,k) = 1`
```
Comment (abdouskamel) [inline code/unclassified/src/josephus_problem/README.md]: Please, put the initialization and the inductive formula in two separated lines.

**Comment by rimshaakhan** [comment]
@abdouskamel Is it alright?

**Comment by rimshaakhan** [comment]
@arnavb @ittempt Can you please review this pull request

**Comment by rimshaakhan** [comment]
@arnavb @iattempt Can you please review this pull request?

**Comment by arnavb** [comment]
@rimshaakhan Just as a suggestion, if you want a maintainer to review, call then by using \@OpenGenus/cosmos-maintainers

File: code/unclassified/src/josephus_problem/README.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+# Josephus Problem
```
Comment (arnavb) [inline code/unclassified/src/josephus_problem/README.md]: You aren't following our documentation guide; All of the markdown should be formatted exactly as outlined in [this article](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md).

The markdown should be (Where you replace important info):

```Markdown
# Algorithm Name
Description

## Explanation
Pictorial diagram to explain the algorithm 

> Image credits: (name of the site/organisation from where the image was referenced)

## Algorithm
Algorithm as a numbered list.


## Complexity
Time and space complexity

---
<p align="center">
	A massive collaborative effort by <a href="https://github.com/OpenGenus/cosmos">OpenGenus Foundation</a> 
</p>

---
```

**Comment by rimshaakhan** [comment]
@arnavb I have made changes. Please review. :smile:

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**